wdctl: add page (#7112)
tldr-pages/tldr: pages/linux/wdctl.md

# wdctl

> > Shows the hardware watchdog status.

Reviewer: ```suggestion
> Show the hardware watchdog status.
```

> More information: <https://manned.org/wdctl>.

- Display the watchdog status:

`wdctl`

> - Display the watchdog status in a single line in `key=value` pairs:

Reviewer 2: I not sure if specific the format in the example description, maybe this is better
```suggestion
- Display the watchdog status in a single line in key-value pairs:
```

# wdctl

> > Shoes hardware watchdog status.

Reviewer 3: ```suggestion
> Shows the hardware watchdog status.
```

> > The Linux kernel can reset the system if serious problems are detected. This can be implemented via special watchdog hardware, or via a slightly less reliable software-only watchdog inside the kernel.

Reviewer 3: This doesn't sound important for this command. Can we remove it?

Author: ```suggestion
```

> > More information: <https://man7.org/linux/man-pages/man8/wdctl.8.html>.

Reviewer 3: ```suggestion
> More information: <https://manned.org/wdctl>.
```

> - Display watchdog status:

Reviewer 3: ```suggestion
- Display the watchdog status:
```

`wdctl`

> - Display watchdog status on a single line (`key=value` pairs):

Reviewer 3: ```suggestion
- Display the watchdog status in a single line in `key=value` pairs:
```

`wdctl --oneline`

> - Display only some watchdog flags (list is driver specific):

Reviewer 3: ```suggestion
- Display only specific watchdog flags (list is driver specific):
```

> `wdctl --flags {{flag list}}`

Reviewer 3: ```suggestion
`wdctl --flags {{flag_list}}`
```
